docs/docusaurus/docs/gx_cloud/tutorials/getting_started/cloud_quickstart.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Quickstart with GX Cloud
+tag: [tutorial, getting started, quickstart, cloud]
+---
+# Quickstart with Great Expectations Cloud
+
+import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+import SetupAndInstallForSqlData from '/docs/components/setup/link_lists/_setup_and_install_for_sql_data.md'
+import SetupAndInstallForFilesystemData from '/docs/components/setup/link_lists/_setup_and_install_for_filesystem_data.md'
+import SetupAndInstallForHostedData from '/docs/components/setup/link_lists/_setup_and_install_for_hosted_data.md'
+import SetupAndInstallForCloudData from '/docs/components/setup/link_lists/_setup_and_install_for_cloud_data.md'
+import Prerequisites from '/docs/components/_prerequisites.jsx'
+
+## Introduction
+
+Few things are as daunting as taking your first steps with a new piece of software. This guide will introduce you to GX Cloud and demonstrate the ease with which you can implement the basic GX workflow. We will walk you through the entire process of connecting to your data, building your first Expectation based off of an initial Batch of that data, validating your data with that Expectation, and finally reviewing the results of your validation.
+
+Once you have completed this guide you will have a foundation in the basics of using GX Cloud. In the future you will be able to adapt GX to suit your specific needs by customizing the execution of the individual steps you will learn here.
+
+## Prerequisites
+
+<Prerequisites>
+
+- Installed Great Expectations OSS on your machine.
+- Followed invitation email instructions from the GX team after signing up for Early Access.
+- Successfully logged in to GX Cloud at [https://app.greatexpectations.io](https://app.greatexpectations.io).
+- A passion for data quality.
+
+</Prerequisites> 
+
+## Overview
+
+With GX Cloud you can get up and running within just a few minutes. The data we're going to use for this overview is the [NYC taxi data](https://www.nyc.gov/site/tlc/about/tlc-trip-record-data.page). This is an open data set which is updated every month. Each record in the data corresponds to one taxi ride. Use individual steps instructions in order to connect to your data.
+The full process you'll be using will look like:
+
+```python title="Jupyter Notebook"
+import great_expectations as gx
+import pandas as pd
+
+
+# Create Data Context
+context = gx.get_context(
+    cloud_access_token = "<user_token_you_generated_in_the_app>",
+    cloud_organization_id = "<organization_id_from_the_app>",
+)
+
+
+# Connect to data
+datasource_name = "my_quickstart_datasource"
+data_connector_name = "default_runtime_data_connector"
+batch_id = "batch_id"
```

Comment:
```suggestion
batch_identifier_name = "path"
data_asset_name = "my_quickstart_table"
```
I think this is clearer. I also think it makes sense to set data_asset_name var so it's more clear that this is user-configurable (should also do this in other section).